### ISSUE TITLE
Unifying Deep Local and Global Features for Efficient Image Search

### ISSUE THREAD

**Issue by chullhwan-song** [chullhwan-song/Reading-Paper#290]
https://arxiv.org/abs/2001.05027

**Comment by chullhwan-song** [comment]
## Abstract
* deep learning 기반 local descriptor와 global descritptor
* 좀 더 디테일하게 말하면 [delf ](https://github.com/chullhwan-song/Reading-Paper/issues/4)+ [gem](https://github.com/chullhwan-song/Reading-Paper/issues/153)결합
* **DELG**, standing for DEep Local and Global features.
* 이 두 개의 feature를 end-to-end learning
* Resnet50 이용
* 기본적인 개념은 global descritptor를 이용하여 찾은 결과를  local descriptor로 re-ranking 하겠다는 개념 
   * 제도 생각했던 개념인데..., 
* 구조
![image](https://user-images.githubusercontent.com/40360823/76508351-93be9e00-6491-11ea-9985-8f2896627f7a.png)
   * 앞에서 언급한 기본 개념
* Global descriptor
   * GEM : 위의 링크 리뷰 참조
![image](https://user-images.githubusercontent.com/40360823/76508531-e5ffbf00-6491-11ea-94b4-7951f1989598.png)
* Local descriptor : delf 위의 링크 참조
* training
![image](https://user-images.githubusercontent.com/40360823/76508654-1a737b00-6492-11ea-8ee7-9175c6437497.png)
   * global descritptor loss
      * GEM 경우에는 arcface loss
   * local descriptor loss
      * Attention Loss  > cross-entropy loss
      * Reconstruction Loss > mean-squared error regression loss
         * feature map(hwc) 끼리의 loss (그림 참조)
      * delf와 다른점은 두번의 softmax loss만 적용했는데...이번에는 end-to-end에 더 촛점을 맞추는듯.
         * 내 생각엔 delf의 첫번째 학습 모델을 이용하여 finetuning한 것을 위의 과정을 그대로 적용하면 어떨지?(논문에 있나?ㅎ)
